package.json

```diff
@@ -120,7 +120,8 @@
     "check:tsd": "tsd --version && tsd",
     "check:dependencies": "mocha test/action/dependency.test.ts",
     "check:dts": "node ./node_modules/typescript/bin/tsc --noEmit mongodb.d.ts && tsd",
-    "check:test": "mocha --config test/mocha_mongodb.json test/integration",
+    "check:search-indexes": "mocha --config test/mocha_mongodb.json test/integration/index-management",
```

Comment:
I think since it will have its separate setup in the future for the Atlas cluster, it probably makes more sense to put this in /test/manual so we don't potentially set up a pattern of individually excluding integration tests in the check:test script.